### PR TITLE
Adding util function to test for depletion

### PR DESCRIPTION
**Pull request by john-science**
## Description

Adding a little helper utility to quickly say "Is depletion being modeled"?

---

## Checklist

- [X] This PR has only one purpose or idea.
- [X] Tests have been added/updated to verify that the new/changed code works.

- [X] The code style follows [good practices](https://terrapower.github.io/armi/developer/standards_and_practices.html).
- [X] The commit message(s) follow [good practices](https://terrapower.github.io/armi/developer/tooling.html).

- [X] The [release notes](https://terrapower.github.io/armi/release/index.html) (location `doc/release/0.X.rst`) are up-to-date with any bug fixes or new features.
- [X] The documentation is still up-to-date in the `doc` folder.
- [X] The dependencies are still up-to-date in `setup.py`.

### CODE REVIEW

File: armi/utils/__init__.py [hunk 1]
```diff
@@ -319,6 +319,21 @@ def getBurnSteps(cs):
     return [len(steps) for steps in stepLengths]
 
 
+def hasBurnup(cs):
+    """Is depletion being modeling?
```
Comment (opotowsky) [inline armi/utils/__init__.py:323]: Grammar: `being modeled?`

**Review by opotowsky — APPROVED** [review]
I'm guessing CI glitched and needs to be rerun. But this is a lovely atomic PR!

**Comment by drewj-usnctech** [comment]
Just chiming in to say thanks 😁

File: armi/utils/__init__.py (hunk 1 above)
Comment (john-science) [reply armi/utils/__init__.py]: Good catch! Fixed.

**Comment by john-science** [comment]
We are having some problem with `coveralls.io`.  The service we use for code coverage reporting.

It is not related to this PR and affects everybody.

I'm trying to fix it. If it doesn't fix soon, I'm going to move us away from [coveralls.io to codecov.io](https://github.com/terrapower/armi/issues/552).

**Pull request merged**